From 76b25271b84cdb0caffbbe9a5f8a17f51c96af4d Mon Sep 17 00:00:00 2001
From: Jeremy Whiting <jpwhiting@kde.org>
Date: Tue, 13 Aug 2013 13:14:20 -0600
Subject: [PATCH 2/4] Fix detection of lzma and xz tar archives downloaded from
 kdelook.org Review: 111952

(cherry picked from commit 60bfd6a58a6ad890ed3fada1d4c8f34ec1ded198)
---
 plasma/package.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plasma/package.cpp b/plasma/package.cpp
index cd3702c..6df4e4e 100644
--- a/plasma/package.cpp
+++ b/plasma/package.cpp
@@ -502,7 +502,7 @@ bool Package::installPackage(const QString &package,
             archive = new KZip(package);
         } else if (mimetype->is("application/x-compressed-tar") ||
                    mimetype->is("application/x-tar")|| mimetype->is("application/x-bzip-compressed-tar") ||
-                   mimetype->is("application/x-xz") || mimetype->is("application/x-lzma")) {
+                   mimetype->is("application/x-xz-compressed-tar") || mimetype->is("application/x-lzma-compressed-tar")) {
             archive = new KTar(package);
         } else {
             kWarning() << "Could not open package file, unsupported archive format:" << package << mimetype->name();
-- 
1.8.3.1

